src/inspector_socket_server.cc

```diff
@@ -16,33 +16,42 @@ namespace inspector {
 // depend on inspector_socket_server.h
 std::string FormatWsAddress(const std::string& host, int port,
                             const std::string& target_id,
-                            bool include_protocol) {
+                            bool include_protocol);
+namespace {
+
+static const uint8_t PROTOCOL_JSON[] = {
+  #include "v8_inspector_protocol_json.h"  // NOLINT(build/include_order)
+};
+
+void Escape(std::string* string) {
+  for (char& c : *string) {
+    c = (c == '\"' || c == '\\') ? '_' : c;
+  }
+}
+
+std::string FormatHostPort(const std::string& host, int port) {
   // Host is valid (socket was bound) so colon means it's a v6 IP address
   bool v6 = host.find(':') != std::string::npos;
   std::ostringstream url;
-  if (include_protocol)
-    url << "ws://";
   if (v6) {
     url << '[';
   }
   url << host;
   if (v6) {
     url << ']';
   }
-  url << ':' << port << '/' << target_id;
+  url << ':' << port;
   return url.str();
 }
 
-namespace {
-
-static const uint8_t PROTOCOL_JSON[] = {
-  #include "v8_inspector_protocol_json.h"  // NOLINT(build/include_order)
-};
-
-void Escape(std::string* string) {
-  for (char& c : *string) {
-    c = (c == '\"' || c == '\\') ? '_' : c;
-  }
+std::string FormatAddress(const std::string& host,
+                            const std::string& target_id,
+                            bool include_protocol) {
```

Comment:
Can you line these two arguments up with the first one.

Reply:
Done